src/EnergyPlus/HVACManager.cc

```diff
@@ -2471,14 +2337,6 @@ void ReportAirHeatBalance(EnergyPlusData &state)
     state.dataHeatBal->ZoneTotalExfiltrationHeatLoss = 0.0;
     state.dataHeatBal->ZoneTotalExhaustHeatLoss = 0.0;
 
-    auto &Zone = state.dataHeatBal->Zone;
-    auto &ZnAirRpt = state.dataHeatBal->ZnAirRpt;
-    auto &Ventilation = state.dataHeatBal->Ventilation;
-    auto &Mixing = state.dataHeatBal->Mixing;
-    auto &CrossMixing = state.dataHeatBal->CrossMixing;
-    auto &RefDoorMixing = state.dataHeatBal->RefDoorMixing;
-    auto &ZoneEquipConfig = state.dataZoneEquip->ZoneEquipConfig;
-    auto &Fan = state.dataFans->Fan;
```

Comment:
[src/EnergyPlus/HVACManager.cc:2474]:(style),[constVariable],Variable 'Zone' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2475]:(style),[constVariable],Variable 'ZnAirRpt' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2476]:(style),[constVariable],Variable 'Ventilation' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2477]:(style),[constVariable],Variable 'Mixing' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2478]:(style),[constVariable],Variable 'CrossMixing' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2479]:(style),[constVariable],Variable 'RefDoorMixing' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2480]:(style),[constVariable],Variable 'ZoneEquipConfig' can be declared as reference to const
[src/EnergyPlus/HVACManager.cc:2481]:(style),[constVariable],Variable 'Fan' can be declared as reference to const